tests/patterns/python/any_qualified_with_wildcard.sgrep

```diff
@@ -0,0 +1 @@
+$X.$Y
```

Comment:
Makes me wonder if we should support this equivalence in the present of metavariables. It could lead to many FPs. Personally I would only apply the equivalence when the dotted ident has no metavariables. If you're looking for `A.foo` and I find `foo` within a `from A import *` I think it's reasonable to match. But I don't think it's reasonable to match e.g. `A.$X` against every unknown identifier just because there is a `from A import *`. I think this is better done in Pro with inter-file naming. I also don't think this is something that SR has really requested?